Formula/theharvester.rb

```diff
@@ -362,6 +362,11 @@ class Theharvester < Formula
   end
 
   def install
+    # Update aiohttp for Python 3.11
+    # https://github.com/laramies/theHarvester/commit/3a800533f45507152f57aa287867f3b5e8f0aa25.patch?full_index=1
+    # Patch does not apply, remove inreplace in next release
+    inreplace "requirements/base.txt", "aiohttp==3.8.1", "aiohttp==3.8.3"
```

Comment:
I don't think this is necessary as I didn't see any source loading of requirements, so I expect it to only be used if user runs `pip install -r [requirements.txt | requirements/base.txt]`.